### DISCUSSION TITLE
How to remove the `deployments_box` from my Pull requests

### DISCUSSION THREAD

**Discussion by ugarciamtz** [community/community#25891] [Actions]
<p>I’m trying to find a way to remove or use the <code>deployments_box</code> (see the attachment) that’s showing up on each PR, but doesn’t have any funcionallity, it used to work when we used <code>Heroku</code>, but not anymore since we move to <code>Netlify</code>, if we can use that box to show our <code>Netlify</code> deployments will be good, if not I would like to get your help on how to remove it</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a9502170d3ff6e9a4f8785cac514190b36c00ca4.png" data-download-href="/uploads/short-url/o9On2mWRJTMfS5NXIJYHZMwTqde.png?dl=1" title="Screen Shot 2019-09-20 at 1.26.35 PM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/51417273/181093838-0221271e-ded9-40a8-a5a2-bb88ef42aebb.png" alt="Screen Shot 2019-09-20 at 1.26.35 PM.png" data-base62-sha1="o9On2mWRJTMfS5NXIJYHZMwTqde" width="690" height="127" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/a9502170d3ff6e9a4f8785cac514190b36c00ca4_2_690x127.png, https://aws1.discourse-cdn.com/github/original/2X/a/a9502170d3ff6e9a4f8785cac514190b36c00ca4.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/a/a9502170d3ff6e9a4f8785cac514190b36c00ca4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a9502170d3ff6e9a4f8785cac514190b36c00ca4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-09-20 at 1.26.35 PM.png</span><span class="informations">856×158 12.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thank you for the help in advance and best regards</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/ugarciamtz">@ugarciamtz</a>,</p>
<p>Thanks for being here! I’m afraid removing the <code>deployments_box</code> is not possible at this time. I’ve taken your feedback and passed it along to the appropriate teams. Thanks again for reaching out :grinning:</p>
